src/components/step-sequence/step-sequence-item/step-sequence-item.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Icon from '../../icon';
+import './step-sequence-item.scss';
+
+const baseClass = 'carbon-step-sequence-item';
+const classes = state => `${baseClass} ${baseClass}--${state}`;
+
+const stepMarker = (state, indicator) => {
+  return state === 'complete' ? <Icon type='tick' /> : indicator;
+};
+
+const StepSequenceItem = ({ children, state, indicator }) => (
+  <li className={ classes(state) }>
```

Comment:
Add `aria-current="step"` to the current step (see https://www.w3.org/TR/wai-aria-1.1/#aria-current)